prime.rs

```diff
@@ -0,0 +1,28 @@
+use std::io;
+fn main() {
+    println!("Enter a Number: ");
+    let mut num = String::new();
+    io::stdin()
+        .read_line(&mut num)
+        .expect("Error reading number");
+    let mut n = num.trim()
+               .parse::<u32>()
+             .unwrap();
+	
+	let mut c=0;

```

Comment:
removed assigned 0. This value is replaced in Line 15.

replace with `let mut c;`